### PR TITLE
write timeout parameter explicitly

### LINKED ISSUE #1854: write timeout explicitly in readiness probe
the [default timeout](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes) for k8s readiness probe is 1s, this may not be enough in some case, should write timeout value explicitly.

### PR DESCRIPTION
**Pull request by xudifsd**
fix #1854

### CODE REVIEW

File: src/job-exporter/deploy/job-exporter.yaml.template [hunk 1]
```diff
@@ -41,6 +41,7 @@ spec:
             port: {{ cluster_cfg["job-exporter"]["port"] }}
           initialDelaySeconds: 3
           periodSeconds: 30
+          timeoutSeconds: 3
```
Comment (YanjieGao) [inline src/job-exporter/deploy/job-exporter.yaml.template]: Does this 3 a little small ? which maybe easy to see readiness probe failed ?

Comment (xudifsd) [reply src/job-exporter/deploy/job-exporter.yaml.template]: The default is 1s. Our bed shows 1s won't be too small.

Comment (fanyangCS) [reply src/job-exporter/deploy/job-exporter.yaml.template]: @xudifsd , please consider setting the value to the worst case.

Comment (xudifsd) [reply src/job-exporter/deploy/job-exporter.yaml.template]: maybe 10?

**Pull request merged**